aces/Loris
# [Data Dictionary] Permission Fix For Description Field

This pull request Resolves [Redmine 14744](https://redmine.cbrain.mcgill.ca/issues/14774)
Without  'Edit permissions for Data Dictionary'  Description field is enabled for editing.

**Note:** The changes that a user make without the edit permission is not updated in the database as ajax call is checking the required permission. But the edit mode is enabled for the description field without any permission checks.

**To Recreate the bug:**
Try removing your  'Edit permissions for Data Dictionary'  and still the Description field is editable in data Dictionary.

**To test the fix with this PR:**
Try removing your permission 'Edit permissions for Data Dictionary'  and Description Field is now no more editable.
Try adding the permission  'Edit permissions for Data Dictionary' again and see if Description Field is editable.
